File: java/05-reactor/src/test/java/com/fResult/reactor/ch5_03/SimpleFluxFactoriesTest.java
import org.junit.jupiter.api.Test;
import org.reactivestreams.Publisher;
import reactor.core.publisher.Flux;
import reactor.test.StepVerifier;
public class SimpleFluxFactoriesTest {
  @Test
  void simple() {
    Publisher<Integer> rangeOfIntegers = Flux.range(0, 10);
    StepVerifier.create(rangeOfIntegers).expectNextCount(10).verifyComplete();
  }
}

File: kotlin/05-reactor/src/test/kotlin/com.fResult/reactor/ch5_03/SimpleFluxFactoriesTest.kt
package com.fResult.reactor.ch5_03

import org.junit.jupiter.api.Test
import org.reactivestreams.Publisher
import reactor.core.publisher.Flux
import reactor.test.StepVerifier

class SimpleFluxFactoriesTest {
  @Test
  fun simple(): Unit {
    val rangeOfIntegers: Publisher<Int> = Flux.range(1, 10)
    StepVerifier.create(rangeOfIntegers).expectNextCount(10).verifyComplete()

    val letters = Flux.just('A', 'B', 'C')
    StepVerifier.create(letters).expectNext('A', 'B', 'C').verifyComplete()
  }
}
